src/resources/messages/interfaces.ts

```diff
@@ -0,0 +1,46 @@
+export interface Message {
+  id: string;
+  channel_id: string;
+  recipient: string | ObjectIdentifier;
+  workflow: string;
+  tenant: string | null;
+  status: string;
```

Comment:
We should probably encode this as a type as well. Like `MessageStatus = "seen" | "read"` etc